lib/data_migrations/form526_submission_submit_endpoint_update.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module DataMigrations
+  module Form526SubmissionSubmitEndpointUpdate
+    module_function
+
+    def run
+      Form526Submission.where(submit_endpoint: nil).in_batches do |batch|
+        # rubocop:disable Rails/SkipsModelValidations
+        batch.update_all(submit_endpoint: :evss)
+        # rubocop:enable Rails/SkipsModelValidations
```

Comment:
Hmm is this an existing precedent for data migrations? I feel like skipping validations is quite dangerous

Reply:
Followed Platform's guidance [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-database-migrations#vets-apidatabasemigrations-DataMigrations) as well as optimizing performance when bulk updating.

Reply:
Hmm I didn't see anything in there about skipping validations am I missing something? Is there a reason why you needed to skip them? At the very least if there is a good reason it would be good to have it as a comment and in the git history even if the file gets removed

Reply:
what does the Validation look like if we added it? I can understand both points here.

Reply:
we know what we are doing. we created a new column, and are updating all the old claims with the correct value.

Reply:
Also, the database steps will need to be split into seperate PRs
https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-database-migrations#vets-apidatabasemigrations-HotCompatibility